src/OpenTelemetry.AutoInstrumentation/Configuration/LogSettings.cs

```diff
@@ -0,0 +1,107 @@
+// <copyright file="LogSettings.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace OpenTelemetry.AutoInstrumentation.Configuration;
+
+/// <summary>
+/// Log Settings
+/// </summary>
+public class LogSettings : Settings
```

Comment:
Other settings have some base tests. See SettingsTests class. I think that this class should be also covered.
If you do not want to do this in this PR, please create separate issue.

Reply:
Yes, it's in my TODO list to create an issue.